Scripts/Prerequisite.ps1

```diff
@@ -0,0 +1,96 @@
+param(
+    [Parameter(Mandatory = $true)]
+    [string]$subscriptionId,
```

Comment:
[Optional] Pascal case for script parameters as a standard for entire project

Reply:
We will go with camel casing as discussed.